### PR TITLE
LPS-168800 LPS-168800Replace liferay liferay-ui:icon-menu with clay:dropdown-menu in edit_asset_list_entry_manual.jsp

### PR DESCRIPTION
**Pull request by DiegoHu97**

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

File: modules/apps/asset/asset-list-web/src/main/resources/META-INF/resources/edit_asset_list_entry_manual.jsp [hunk 1]
```diff
@@ -110,33 +110,18 @@ AssetListEntry assetListEntry = assetListDisplayContext.getAssetListEntry();
...
-									%>
-
-								</liferay-ui:icon-menu>
+								<%
+								EditAssetListEntryManualActionDropdownItemsProvider editAssetListEntryManualActionDropdownItemsProvider = new EditAssetListEntryManualActionDropdownItemsProvider(editAssetListDisplayContext);
+								%>
+
+								<clay:dropdown-menu
+									aria-label='<%= LanguageUtil.get(request, "show-actions") %>'
+									cssClass="btn btn-secondary btn-sm"
```
Comment (victorg1991) [inline modules/apps/asset/asset-list-web/src/main/resources/META-INF/resources/edit_asset_list_entry_manual.jsp:120]: I believe here it should be something like select-items or something like that

File: modules/apps/asset/asset-list-web/src/main/resources/META-INF/resources/js/EditAssetListEntryManualDefaultPropsTransformer.js [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+	actions,
+	items,
+	portletNamespace,
+	...props
+}) {
+	const updateItem = (item) => {
+		const newItem = {
+			...item,
+			onClick(event) {
+				if (item.data.title && item.data.href) {
```
Comment (victorg1991) [inline modules/apps/asset/asset-list-web/src/main/resources/META-INF/resources/js/EditAssetListEntryManualDefaultPropsTransformer.js:27]: title and href is available in all the actions right? can we remove this if?

File: modules/apps/asset/asset-list-web/src/main/resources/META-INF/resources/js/EditAssetListEntryManualDefaultPropsTransformer.js [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+					openSelectionModal({
+						customSelectEvent: true,
+						multiple: true,
+						onSelect(data) {
+							if (data.value && data.value.length) {
+								const selectedItems = data.value;
+								const assetEntryIds = [];
+
+								Array.prototype.forEach.call(
+									selectedItems,
```
Comment (victorg1991) [inline modules/apps/asset/asset-list-web/src/main/resources/META-INF/resources/js/EditAssetListEntryManualDefaultPropsTransformer.js:39]: Array.from(selectedItems).map will be better here

Something like:

```javascript
Array.from(selectedItems).map(selectedItem => {
 const assetEntry = JSON.parse(
     selectedItem 
   );
 return assetEntry.assetEntryId;
}).join(',')
```

**Comment by ealonso** [comment]
Just started reviewing :)

**Comment by ealonso** [comment]
see: https://github.com/liferay-echo/liferay-portal/pull/10353

**Pull request closed without merging**